source/main/physics/Beam.cpp

```diff
@@ -4498,13 +4496,14 @@ void Beam::updateDebugOverlay()
 
 void Beam::updateNetworkInfo()
 {
+	if (!gEnv->network) return;

```

Comment:
No no _no_ NO! :angry: 
This is just so damn wrong. It's our codebase, our function, yet we're unable to call it right, so we check with a global state? Sorry but this is one of the worst kinds of bullshit RoR is plagued with. Worse yet, it comes in a **[cleanup]** commit ... seriously?